documentation/modules/operators/ref-operator-cluster-feature-gate-releases.adoc

```diff
@@ -17,9 +17,9 @@ Beta stage features are well tested and their functionality is not likely to cha
 GA stage features are stable and should not change in the future.
 Alpha and beta stage features are removed if they do not prove to be useful.
 
-* The `ControlPlaneListener` feature gate moved to beta stage in Strimzi 0.27 and is expected to move to general availability in Strimzi 0.32.
+* The `ControlPlaneListener` feature gate moved to GA stage in Strimzi 0.32. The control plane listener is now always used.
```

Comment:
Don't we want to use the same wording as `ServiceAccountPatching`? _It is now permanently enabled and cannot be disabled._